src/io/flutter/view/FlutterView.java

```diff
@@ -237,6 +238,16 @@ protected void perform(AnActionEvent event) {
   }
 }
 
+class ToggleInspectModeAction extends AbstractToggleableAction {
```

Comment:
Ah, ok. I had imagined that there was an on-device way to exit inspector mode, and that a 'toggle' button in IntelliJ, w/o an event or such from the device, would get out of sync.